addons/web_editor/models/ir_qweb.py

```diff
@@ -302,7 +302,9 @@ class Image(models.AbstractModel):
 
     @api.model
     def from_html(self, model, field, element):
-        url = element.find('img').get('src')
+        if element.find('.//img') is None:
```

Comment:
It is mandatory to change the xpath selector?
Eg why `find('img')` became `find('.//img')`?

Reply:
It is due to copy paste from master branch I created previously ;)